OpenEdXMobile/src/main/java/org/edx/mobile/util/FileUtil.java

```diff
@@ -260,7 +260,9 @@ public static void deleteRecursive(@NonNull File fileOrDirectory,
      * @param filepath Video storage file path
      * @return True if video file exists on a given path
      */
-    public static boolean isVideoFileExists(@NonNull Context context, @NonNull String filepath) {
+    public static boolean isVideoFileExists(@NonNull Context context, @Nullable String filepath) {
+        if (TextUtils.isEmpty(filepath))
```

Comment:
If you are marking it nullable then please add a null check as well.

Reply:
`TextUtils.isEmpty` => Returns true if the string is null or 0-length.